README.md

```diff
@@ -153,6 +152,20 @@ Signed-Releases: Fail 0
 Signed-Tags: Fail 10
 ```
 
+### Caching
+
+Scorecard uses `httpcache` with <https://docs.github.com/en/rest/overview/resources-in-the-rest-api#conditional-requests> for caching httpresponse. The default cache is in-memory.
```

Comment:
Maybe link to the etags stuff in GitHub - the real benefit is avoiding the API quota